.github/CONTRIBUTING.md

```diff
@@ -1,64 +1,65 @@
+[issue]: https://github.com/PlaceholderAPI/PlaceholderAPI/issues/new
+[discord]: https://helpch.at/discord
+[Code of Conduct]: https://github.com/PlaceholderAPI/PlaceholderAPI/blob/master/CODE_OF_CONDUCT.md
+[wiki]: https://github.com/PlaceholderAPI/PlaceholderAPI/blob/master/wiki
 
-# Contributing
+# Contributing Guidelines
+We welcome everyone to contribute towards the PlaceholderAPI Project, but doing so will require you to follow specific rules to keep a consistent and welcoming way of contributing.
 
-When contributing to this repository, please first discuss the change you wish to make via [issue] or through [Discord] with the owners of this repository before making a change. 
+If you have any questions about this contributing file, feel free to contact us on our [Discord Server][discord].
 
-Please note we have a code of conduct, please follow it in all your interactions with the project.
+## Issues
+Like any other Project can you encounter bugs or a feature is missing for you in the plugin.  
+For that can you open an [issue] to report a bug or suggest a new feature to be added.
 
-## Pull Request Process
+When doing so, make sure you follow the below rules.
 
-1. Update the README.md with details of changes if it is affected by those changes.
-2. Try to prevent any breaking changes affecting the end-user (Developers using the API).  
-Mention any breaking changes that could affect the end-user.
-3. Document any public methods that the end-user might use.
+### Follow the Template
+We have Issue templates to help us get the required information more easly. Please follow the provided template when either filing a bug report or feature request.  
+We reserve the right to close your issue without warning for not following the templates.
 
-## Code of Conduct
-
-### Our Pledge
-
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
-
-### Our Standards
+### Using Latest Version
+When it comes to bug reports should you always check first, if you're using the latest release of PlaceholderAPI.  
+Often is a bug you encounter fixed in a newer release.
 
-Examples of behavior that contributes to creating a positive environment
-include:
+You can optionally also try out development builds to see, if your bug is fixed in those.
 
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
+The same rules apply for when making a feature request.
 
-Examples of unacceptable behavior by participants include:
+### No duplicate issue
```

Comment:
Why do ### titles suddenly stop being capitalised here? Not that it's bad, but it should be consistent. Decide on a single style. Personally, I prefer this form, with only the first word capitalised.

Reply:
Most likely due to my german grammar knowledge where words prefixed with "the" (Nouns I suppose) are always capitalized.